tests/roxctl/bats-tests/local/roxctl-netpol-generate-development.bats

```diff
@@ -115,6 +115,28 @@ teardown() {
     assert_line --index 5 'doc: 2'
 }
 
+@test "roxctl-development netpol generate generates network policies with custom dns named port" {
+    assert_file_exist "${test_data}/np-guard/scenario-minimal-service/frontend.yaml"
+    assert_file_exist "${test_data}/np-guard/scenario-minimal-service/backend.yaml"
+    echo "Writing network policies to ${ofile}" >&3
+    dns_port="dns"
+    run roxctl-development netpol generate "${test_data}/np-guard/scenario-minimal-service" --dnsport ${dns_port}
```

Comment:
I would like us to also test the two other combinations that may lead to potential errors:

```
run roxctl-development netpol generate "${test_data}/np-guard/scenario-minimal-service" --dnsport ""`
```
and where `--dnsport` is not used at all (I think, this may be already in place).

Reply:
Added a test with `--dnsport` set to the empty string.
Good-path tests which do not specify `--dnsport` already exist.

Reply:
Is `--dnsport` a required parameter? If not, I think a test without the explicit `--dnsport` might be a good addition:
`run roxctl-development netpol generate "${test_data}/np-guard/scenario-minimal-service"`

Reply:
`--dnsport` is not a required parameter.
The bat test "roxctl-development netpol generate generates network policies" runs without `--dnsport`.